recipes/tiny-aes-c/all/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.4)
+project(cmake_wrapper)
+
+if(WIN32 AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+endif()
```

The if is not required

```suggestion
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```